Makefile

```diff
@@ -196,6 +196,7 @@ inplace-conf-common: dist
 	            --with-domserver_tmpdir=$(CURDIR)/output/tmp \
 	            --with-judgehost_tmpdir=$(CURDIR)/output/tmp \
 	            --with-judgehost_judgedir=$(CURDIR)/output/judgings \
+		    --with-domserver_databasedumpdir=$(CURDIR)/output/db-dumps \
```

Comment:
indentation doesn't match (at least in github preview)

Reply:
I agree, I fixed that already in https://github.com/DOMjudge/domjudge/pull/2184

Reply:
Does this mean this PR is now obsolete? Or needs to be updated?

Reply:
This means that I first want that other one merged and than work on the more difficult part namely the extra `sudo` needed etc.